### DISCUSSION TITLE
Support auto-completing in SQL Editor

### DISCUSSION THREAD

**Discussion by OmarBazaraa** [grafana/clickhouse-datasource#750] [Ideas]
It'd be really handy if the plugin supports auto-completing when typing the names of databases, tables, columns, and functions.

**Comment by SpencerTorres** [comment]
I believe there is some auto-complete built in, at least for the table names. It might be more complicated to pull some of the other information though. I don't recall how it's implemented now, but we definitely don't want to load every schema from every database/table + all functions every time the editor is opened. Maybe it can be reduced, like after a `FROM table` is written.
